packages/file_selector/file_selector_linux/.gitignore

```diff
@@ -3,3 +3,7 @@
 .flutter-plugins
 .flutter-plugins-dependencies
 pubspec.lock
+
+# Linux specific
```

Comment:
Linux-specific ignores belong in `linux/.gitignore`, not the top-level `.gitignore`.

Why are these needed though? What is creating them? I've never seen these locally.

Reply:
Those are being created when running the example app in linux